src/app/app.component.ts

```diff
@@ -1,10 +1,17 @@
-import { Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouterLink, RouterOutlet } from "@angular/router";
+import { LoginComponent } from "./features/auth/login/login.component";
+import { HomeComponent } from "@src/app/features/home/home.component";
+import { SpinnerComponent } from "@src/app/shared/components/spinner/spinner.component";
+import { CommonModule } from "@angular/common";
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
+  standalone: true,
+  imports: [RouterOutlet, RouterLink, LoginComponent, HomeComponent, SpinnerComponent, CommonModule],
```

Comment:
Only `RouterOutlet` is being used in this component. So other imports are not necessary